src/backend/src/controllers/reimbursement-requests.controllers.ts

```diff
@@ -33,4 +33,15 @@ export default class ReimbursementRequestController {
       next(error);
     }
   }
+
+  static async createExpenseType(req: Request, res: Response, next: NextFunction) {
+    try {
+      const { name, code, allowed } = req.body;
+      const user = await getCurrentUser(res);
+      const createdExpenseTypeId = await ReimbursementRequestService.createExpenseType(user, name, code, allowed);
```

Comment:
return object instead of id, make sure to update documentation in service method